src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -719,6 +729,12 @@ private void SearchPackagesAndRefreshUpdateCount(bool useCacheForUpdates)
             .FileAndForget(TelemetryUtility.CreateFileAndForgetEventName(nameof(PackageManagerControl), nameof(SearchPackagesAndRefreshUpdateCount)));
         }
 
+        public bool IsRecommenderFlightEnabled()
+        {
+            // TODO: Need to use correct flight name
+            return ExperimentationService.Default.IsCachedFlightEnabled("nugetrecommendpkgs");
```

Comment:
Does this make a network call? 
Any threading affinity?

Reply:
This is currently happening on the UI thread and that can be problematic.

Reply:
It doesn’t make a network call.  It reads information from local storage and is intended to be fast and cheap.